src/index.ts

```diff
@@ -189,7 +205,7 @@ const ts2gas = (source: string, transpileOptions: ts.TranspileOptions = {}) => {
   // ## Exports
   // Exports are transpiled to variables 'exports' and 'module.exports'
 
-  const pjson = require('../package.json');
+  const pjson = require('../package.json');  // ugly hack
```

Comment:
I don't think this is really ugly, besides the relative require.

Reply:
it's more because lots of statements stand here and there without e clear structure/design. Like the types and interfaces I added